yellr-serv/yellrserv/ep_client_post.py

```diff
@@ -0,0 +1,244 @@
+import os
+import json
+import urllib
+import uuid
+
+#import subprocess
+#import magic
+#import mutagen.mp3
+#import mutagen.oggvorbis
+#import mutagen.mp4
+
+from pyramid.view import view_config
+
+import client_utils
+import utils
+from config import system_config
+
+@view_config(route_name='get_local_posts.json')
+def get_local_posts(request):
+
+    result = {'success': False}
+    status_code = 200
+
+    try:
+    #if True:
+
+        success, error_text, language_code, lat, lng, \
+            client = client_utils.register_client(request)
+        if success == False:
+            raise Exception(error_text)
+
+        start = 0
+        count = 50
+        try:
+            if 'start' in reqeusts.GET:
+                start = int(float(request.GET['start']))
+            if 'count' in request.GET:
+                count = int(float(request.GET['count']))
+        except:
```

Comment:
in this case, the try/except is catching an invalid number within the start or count GET params.  We set defaults for each, so there is no reason to do anything with the exception.  I see no reason to not do it this way, however if you do not want to see it, the start and count defaults could be placed within the except.

I think this is a completely acceptable way to do this though.